comp/core/status/statusimpl/common_header_provider.go

```diff
@@ -68,24 +67,24 @@ func (h *headerProvider) HTML(buffer io.Writer) error {
 	return t.Execute(buffer, h.data)
 }
 
-func newCommonHeaderProvider(config config.Component) status.HeaderProvider {
+func newCommonHeaderProvider(params status.Params, config config.Component) status.HeaderProvider {
 
 	data := map[string]interface{}{}
 	data["version"] = version.AgentVersion
-	data["flavor"] = flavor.GetFlavor()
```

Comment:
Is `GetFlavorr` and `GetHumanReadableFlavor` the same ?

Reply:
It is not the same. I was thinking of cutting corners and only use `GetHumanReadableFlavor` when populating the params